### PR TITLE
feat: translation zh.json 

### PR DESCRIPTION
**Pull request by juliahermak**
Screenshot 2024-01-10 23 50 10
Screenshot 2024-01-10 23 49 46

### CODE REVIEW

**Comment by juliahermak** [comment]
[Translate the app to Chinese language](https://trello.com/c/ox0upUkP/481-translate-the-app-to-chinese-language)

File: src/locales/zh.json [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      "exchange_bot": "嗨！我是你的交换机器人₿. 我立即匿名操作。键入**/help**查看我能做些什么**ℹ**️ 在ADAMANT博客上了解更多信息。",
+      "exchange_bot_title": "ADAMANT Exchange",
+      "bounty_bot": "欢迎！通过您的社交活动获得ADM！发送**/help**命令了解详细信息。",
+      "bounty_bot_title": "ADM Bounty bot",
+      "donate_bot": "感谢您的支持，我们开发了去中心化的ADAMANT Messenger——在此聊天中转移ADM、BTC、ETH和其他硬币。有关其他捐赠钱包，请参阅https://adamant.im/donate.",
+      "donate_bot_title": "向ADAMANT基金会捐款",
+      "welcome_message": "ADAMANT是一种独特的区块链信使，独立于政府、企业甚至其开发者。这是由于去中心化的网络、完全开放的源代码和用户的反馈。这就是为什么每一个动作，包括消息传递或保存新联系人，都要收取0.001 ADM的网络费用。\n\n与典型的P2P和集中式信使不同，区块链系统提供了卓越的安全性y和隐私。更重要的是，它为用户提供了新的可能性，例如在聊天中存储和传输加密货币，并完全控制私钥；使用ADAMANT作为2FA解决方案，匿名交换加密货币，并押注比特币利率等。\\n\nADAMANT区块链系统属于其用户。没有人可以控制、阻止、停用、限制或审查帐户。用户对其内容、消息、媒体以及使用信使的目标和意图承担全部责任。\\n\nDAMANT是匿名的，您的IP甚至对合作伙伴都是隐藏的，Tor版本可供那些想将隐私提升到一个新水平的人使用。但请记住，你的安全和匿名取决于你。不要关注你收到的链接，否则你的IP可能会被泄露。在设备上设置密码。了解有关安全性和匿名性的更多信息，请访问https://adamant.im/staysecured/.\\n\n请确保您已为此帐户保存了唯一的密码短语--注销并再次登录。把你的密码也写在纸上。只有*您对密码短语的安全负责*。它无法恢复。如果其他人得到了它，你的代币就会被盗，你的信件也会被泄露。把这件事看得最重要，就好像你钱包里的代币要花10亿美元一样。\\n\n要立即开始发送消息，请在“帐户”选项卡上获得免费欢迎令牌。然后开始新的聊天并填写朋友的ADM地址。建议您仅通过ADAMANT个人共享ADM地址。\\n\n在Internet上搜索ADAMANT机器人程序的地址。他们可以交换加密货币，接受对加密货币利率的押注，并实现愿望。\n",
```
Comment (adamant-al) [inline src/locales/zh.json:68]:
```suggestion
      "welcome_message": "ADAMANT是一种独特的区块链信使，独立于政府、企业甚至其开发者。这是由于去中心化的网络、完全开放的源代码和用户的反馈。这就是为什么每一个动作，包括消息传递或保存新联系人，都要收取0.001 ADM的网络费用。\n\n与典型的P2P和集中式信使不同，区块链系统提供了卓越的安全性y和隐私。更重要的是，它为用户提供了新的可能性，例如在聊天中存储和传输加密货币，并完全控制私钥；使用ADAMANT作为2FA解决方案，匿名交换加密货币，并押注比特币利率等。\n\nADAMANT区块链系统属于其用户。没有人可以控制、阻止、停用、限制或审查帐户。用户对其内容、消息、媒体以及使用信使的目标和意图承担全部责任。\n\nADAMANT是匿名的，您的IP甚至对合作伙伴都是隐藏的，Tor版本可供那些想将隐私提升到一个新水平的人使用。但请记住，你的安全和匿名取决于你。不要关注你收到的链接，否则你的IP可能会被泄露。在设备上设置密码。了解有关安全性和匿名性的更多信息，请访问https://adamant.im/staysecured/ \n\n请确保您已为此帐户保存了唯一的密码短语--注销并再次登录。把你的密码也写在纸上。只有*您对密码短语的安全负责*。它无法恢复。如果其他人得到了它，你的代币就会被盗，你的信件也会被泄露。把这件事看得最重要，就好像你钱包里的代币要花10亿美元一样。\n\n要立即开始发送消息，请在“帐户”选项卡上获得免费欢迎令牌。然后开始新的聊天并填写朋友的ADM地址。建议您仅通过ADAMANT个人共享ADM地址。\n\n在Internet上搜索ADAMANT机器人程序的地址。他们可以交换加密货币，接受对加密货币利率的押注，并实现愿望。",
```

File: src/locales/zh.json [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+      "REGISTERED": "传递到节点",
+      "INVALID": "信息不正确。请仔细检查区块链浏览器中的Tx",
+      "PENDING": "尚未确认。正在获取…",
+      "REJECTED": "无法获取Tx信息。它可能被取消或不被接受。请在区块链资源管理器中检查Tx",
+      "UNKNOWN": "此加密货币尚不受支持"
+    },
+    "virtual": {
+      "bitcoin_bet": "嗨！我是匿名的，区块链证明了赌注机器人🤑. 我接受BTC费率的投注，并向中奖者支付奖励。在ADAMANT的博客上了解更多信息，或键入**/help**开始下注。",
+      "bitcoin_bet_title": "押注比特币价格",
+      "adelina": "你好，我是adelina。让我们愉快地交谈一下！我可以做很多事情，比如：\n-起草一封电子邮件或其他文章\n-编写Python代码\n-回答关于一组文档的问题\n-支持对话\n-一系列主题的导师\n-翻译语言\n-写诗\n-模拟电子游戏中的角色等等",
```
Comment (adamant-al) [inline src/locales/zh.json:58]:
```suggestion
      "adelina": "你好，我是Adelina。让我们愉快地交谈一下！我可以做很多事情，比如：\n-起草一封电子邮件或其他文章\n-编写Python代码\n-回答关于一组文档的问题\n-支持对话\n-一系列主题的导师\n-翻译语言\n-写诗\n-模拟电子游戏中的角色等等",
```

File: src/locales/zh.json [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+  "home": {
+    "balance": "余额",
+    "buy_tokens_anonymously": "在网站上匿名",
+    "buy_tokens_exchanger": "聊天内交换器",
+    "copied": "已复制",
+    "copy": "复制",
+    "copy_address": "复制地址",
+    "copy_uri": "复制共享链接",
+    "create_qrcode": "创建二维码",
+    "free_adm_btn": "释放adm令牌",
```
Comment (adamant-al) [inline src/locales/zh.json:98]:
```suggestion
    "free_adm_btn": "释放ADM令牌",
```

File: src/locales/zh.json [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+    "user_password_hint": "用户密码",
+    "user_password_title": "解锁密码",
+    "user_password_unlock": "解锁"
+  },
+  "nodes": {
+    "active": "活动",
+    "fastest_title": "选择最快的节点",
+    "fastest_tooltip": "将使用最快的节点处理请求",
+    "tabs": {
+      "adm_nodes": "adm节点",
```
Comment (adamant-al) [inline src/locales/zh.json:163]:
```suggestion
      "adm_nodes": "ADM节点",
```

File: src/locales/zh.json [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+    "inactive": "Inactive",
+    "sync": "同步",
+    "ms": "ms",
+    "nodeLabelDescription": "支持去中心化并提高隐私级别--<a target=\"_blank\" href=\"https://medium.com/adamant-im/how-to-run-your-adamant-node-on-ubuntu-990e391e8fcc\">运行您自己的ADAMANT节点</a>。若要将节点添加到列表中，您需要<a target=\"_blank\"href=\"https://github.com/Adamant-im/adamant-im/\">在单独的域上部署web应用程序</a>。此限制指的是内容安全策略（CSP）。",
+    "offline": "脱机",
+    "ping": "Ping",
+    "socket": "套接字",
+    "unsupported": "不受欢迎",
+    "unsupported_reason_protocol": "不允许HTTP",
+    "unsupported_reason_api_version": "api版本过期",
```
Comment (adamant-al) [inline src/locales/zh.json:177]:
```suggestion
    "unsupported_reason_api_version": "API版本过期",
```

File: src/locales/zh.json [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+    "language_label": "语言",
+    "currency_label": "主货币",
+    "nodes_list": "节点列表",
+    "notification_title": "通知",
+    "page_title": "设置",
+    "push_denied": "推送通知在浏览器设置中被拒绝",
+    "push_not_supported": "此浏览器不支持推送通知",
+    "security_title": "安全和登录",
+    "send_on_enter": "按enter键发送",
+    "send_on_enter_tooltip": "用enter发送消息，用Ctrl+enter换行",
```
Comment (adamant-al) [inline src/locales/zh.json:216]:
```suggestion
    "send_on_enter_tooltip": "用Enter发送消息，用Ctrl+Enter换行",
```

File: src/locales/zh.json [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+    "crypto": "加密货币",
+    "decode_from_camera": "用相机扫描二维码",
+    "decode_from_image": "从图像中读取QR",
+    "error_chat_fee": "要通过聊天发送｛crypto｝，您需要0.001 ADM",
+    "error_dust_amount": "无存量--发送更多代币",
+    "recipient_minimum_balance": "收件人必须至少有0.05 LSK--发送更多令牌",
+    "error_precision": "精度过高--设置小数位数较少",
+    "error_erc20_fee": "您需要｛费｝来支付转会费",
+    "error_field_is_required": "字段是必需的",
+    "error_incorrect_address": "crypto｝钱包地址不正确",
```
Comment (adamant-al) [inline src/locales/zh.json:318]:
```suggestion
    "error_incorrect_address": "{crypto｝钱包地址不正确",
```

File: src/locales/zh.json [hunk 8]
```diff
@@ -0,0 +1,380 @@
...
+    "amount_percent_50": "发送一半",
+    "amount_percent_100": "全部发送",
+    "balance": "余额",
+    "comments_label": "注释",
+    "textdata_label": "｛crypto｝区块链中的公共消息",
+    "commission_label": "转让费",
+    "commission_estimate_label": "预计转会费",
+    "confirm_approve": "确认",
+    "confirm_cancel": "取消",
+    "confirm_message": "确认将<b>amount｝｛crypto｝</b>转移到地址<span class=\"address-in-confirm\">{address}</span>.",
```
Comment (adamant-al) [inline src/locales/zh.json:306]:
```suggestion
    "confirm_message": "确认将<b> {amount｝｛crypto｝</b>转移到地址<span class=\"address-in-confirm\">{address}</span>.",
```

File: src/locales/zh.json [hunk 9]
```diff
@@ -0,0 +1,380 @@
...
+    "error_same_recipient": "不要向自己发送令牌",
+    "error_transfer_send": "完成事务时出错",
+    "error_unknown": "未知错误。弱连接？",
+    "error_no_hash": "失败。无法获取Tx哈希",
+    "final_amount_label": "包含转会费的金额",
+    "increase_fee": "增加费用",
+    "invalid_qr_code": "二维码不包含地址或无法识别",
+    "no_address_text": "此用户还没有公共｛crypto｝钱包。他们必须登录Messenger，余额超过0.001 ADM",
+    "no_address_title": "R收件人还没有{crypto}钱包",
+    "legacy_address_text": "此用户有一个旧的｛crypto｝钱包。他们必须登录Messenger，余额超过0.001 ADM才能更新｛crupto｝的钱包地址",
```
Comment (adamant-al) [inline src/locales/zh.json:333]:
```suggestion
    "legacy_address_text": "此用户有一个旧的｛crypto｝钱包。他们必须登录Messenger，余额超过0.001 ADM才能更新｛crypto｝的钱包地址",
```

File: src/locales/zh.json [hunk 10]
```diff
@@ -0,0 +1,380 @@
...
+    "legacy_address_title": "收件人还有一个旧的{crypto}钱包",
+    "send_button": "发送",
+    "to_address_label": "收件人",
+    "to_name_label": "收件人｛name｝",
```
Comment (adamant-al) [inline src/locales/zh.json:337]:
```suggestion
    "to_name_label": "收件人｛name｝",
```

**Review by adamant-al — CHANGES_REQUESTED** [review]

File: src/locales/zh.json [hunk 11]
```diff
@@ -0,0 +1,380 @@
...
+      "modal_header": "扫描包含密码短语的二维码"
+    },
+    "new-chat": {
+      "modal_header": "扫描包含ADAMANT地址的二维码"
+    },
+    "no_camera_found": "未找到摄像头",
+    "something_wrong": "出了问题",
+    "waiting_camera": "正在等待相机…"
+  },
+  "title": "英语",
```
Comment (adamant-al) [inline src/locales/zh.json:246]:
```suggestion
  "title": "中文",
```

File: src/locales/zh.json (hunk 10 above)
Comment (bludnic) [reply src/locales/zh.json:337]: Your suggestion is identical?

**Comment by adamant-al** [comment]
@juliahermak 
Please check for mistakes like `｛crypto}` once more.

**Review by adamant-al — APPROVED** [review]

**Comment by juliahermak** [comment]
@adamant-al I corrected all the wrong characters that I found.

**Comment by adamant-al** [comment]
@juliahermak 
I think you are doing wrong with commas and { }.
In Chinese, punctuation symbols go with spaces as a single character.
So you better revert this commit.

**Review by bludnic — APPROVED** [review]

**Comment by bludnic** [comment]
Chinese `{}` chars don't work when used as a variables like `{crypto}`, `{amount}`. So the chars were replaced by a regular `{}` (without spaces).

@adamant-al probably better to ask Chen. Can he make a suggestions in this PR?

**Comment by adamant-al** [comment]
> Chinese `{}` chars don't work when used as a variables like `{crypto}`, `{amount}`. So the chars were replaced by a regular `{}` (without spaces).
> 
> @adamant-al probably better to ask Chen. Can he make a suggestions in this PR?

Not necessary to ask him.
Use regular { } and spaces around.

**Pull request merged**